src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -858,6 +858,28 @@ class SystemDictionaryDCmd : public DCmdWithParser {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
+class ClassesDCmd : public DCmdWithParser {
+protected:
+  DCmdArgument<bool> _verbose;
+public:
+  ClassesDCmd(outputStream* output, bool heap);
+  static const char* name() {
+    return "VM.classes";
+  }
+  static const char* description() {
+    return "Prints list of all loaded classes";
```

Comment:
I think it's better to say "Print all loaded classes". Most commands in this file do not use the third-person singular verb ending. The words "list of" are redundant.